### PR TITLE
Generate hash file for all installed packages

### PR DESCRIPTION
**Pull request by afbjorklund**
Try to generate the manifest (unidentifiably called "SHA256SUMS"), for the installed packages.

This is a proof-of-concept, for later using `repro-get` to install the packages from the cache...

Currently it comes with some **heavy** dependencies, for dockerconfig and for compression.

Maybe one can work with repro-get upstream, to get those included more on a selective basis ?

Issue:
* #1422

### CODE REVIEW

**Comment by afbjorklund** [comment]
ubuntu:
118M	/var/lib/apt
82M	/var/cache/apt

fedora:
4.2M	/var/lib/dnf
253M	/var/cache/dnf

File: cmd/lima-guestagent/distro_linux.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"sort"
+
+	"github.com/reproducible-containers/repro-get/pkg/distro"
+	"github.com/reproducible-containers/repro-get/pkg/distro/alpine"
+	"github.com/reproducible-containers/repro-get/pkg/distro/arch"
+	"github.com/reproducible-containers/repro-get/pkg/distro/debian"
+	"github.com/reproducible-containers/repro-get/pkg/distro/distroutil/detect"
+	"github.com/reproducible-containers/repro-get/pkg/distro/fedora"
+	"github.com/reproducible-containers/repro-get/pkg/distro/none"
+	"github.com/reproducible-containers/repro-get/pkg/distro/ubuntu"
```
Comment (AkihiroSuda) [inline cmd/lima-guestagent/distro_linux.go:14]: I'd prefer to just os/exec it

File: go.mod [hunk 2]
```diff
@@ -45,20 +46,26 @@ require (
...
 	github.com/a8m/envsubst v1.4.2 // indirect
 	github.com/alecthomas/participle/v2 v2.0.0-beta.5 // indirect
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
+	github.com/containerd/nerdctl v0.23.1-0.20221008120401-b4c01094b581 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/digitalocean/go-libvirt v0.0.0-20201209184759-e2a69bcd5bd1 // indirect
 	github.com/dimchansky/utfbom v1.1.1 // indirect
+	github.com/docker/cli v20.10.18+incompatible // indirect
+	github.com/docker/docker v20.10.18+incompatible // indirect
+	github.com/docker/docker-credential-helpers v0.7.0 // indirect
```
Comment (AkihiroSuda) [inline go.mod:61]: Too many deps. Let's just os/exec it.

**Comment by afbjorklund** [comment]
I guess that means that we have to install the `repro-get` binary (11M), somehow.

File: cmd/lima-guestagent/distro_linux.go (hunk 1 above)
Comment (afbjorklund) [reply cmd/lima-guestagent/distro_linux.go:14]: Maybe this particular function should be moved from cmd to pkg, upstream ? (in repro-get)

**Comment by afbjorklund** [comment]
And it needs to handle errors, for those distributions that are not yet supported.

**Comment by afbjorklund** [comment]
This will **not** bundle partial commands of `github.com/reproducible-containers/repro-get` in Lima, so will close PR.

If it is implemented, it will probably install and use `repro-get` as a program instead. So only the shell part remains...

**Pull request closed without merging**